src/styles/albumsList.styl

```diff
@@ -20,6 +20,7 @@
     .pho-album-photo-item
         height 4em
         width 4em
```

Comment:
I think it still is, otherwise the image will take it's natural width, won't it?